prisma/migrations/20240416013813_add_blocked_workers/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- CreateTable
+CREATE TABLE "BlockedWorker" (
+    "shiftUuid" TEXT NOT NULL,
+    "workerUuid" TEXT NOT NULL,
+    "facilityUuid" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL,
+    "blockReason" TEXT NOT NULL,
+
+    CONSTRAINT "BlockedWorker_pkey" PRIMARY KEY ("facilityUuid","workerUuid")
+);
+
```

Comment:
- Consider adding an index on (workerUuid, facilityUuid) for faster lookups when checking if a worker is blocked.
- Consider adding updatedAt field to BlockedWorker model for better audit tracking.
- Consider adding a status field to BlockedWorker to enable soft deletes instead of hard deletes

Reply:
there is no requirement to delete a blocked worker. all these indexing and all required for performance enhancements, which are not currently in the scope of requirements